src/collective/nitf/upgrades/v2003/__init__.py

```diff
@@ -17,13 +18,19 @@ def remove_searchabletext_metadata(setup_tool):
     logger.info('Found {0} objects'.format(len(results)))
     n = 0
     catalog_object = portal_catalog.catalog_object
+    msg_prefix = 'collective.nitf upgradeStep 2003:'
     for obj in get_valid_objects(results):
         catalog_object(obj, idxs=['id'])
         n += 1
         if n % 1000 == 0 and not test:
+            msg = '{0} items processed.'.format(n)
+            transaction_note('{0} {1}'.format(msg_prefix, msg))
```

Comment:
Please add in the change note that this information is only available in the root undo.